cms/static/js/views/active_video_upload_list.js

```diff
@@ -204,27 +227,77 @@ define([
             },
 
             fileUploadDone: function(event, data) {
-                this.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
-                this.setProgress(data.cid, 1);
-                if (this.onFileUploadDone) {
-                    this.onFileUploadDone(this.collection);
-                    this.clearSuccessful();
-                }
+                var model = this.collection.get(data.cid),
+                    self = this;
+
+                this.readMessages([
+                    StringUtils.interpolate(
+                        gettext('Upload completed for video {fileName}'),
+                        {fileName: model.get('fileName')}
+                    )
+                ]);
+
+                this.sendStatusUpdate([
+                    {
+                        edxVideoId: model.get('videoId'),
+                        status: 'upload_completed',
+                        message: 'Uploaded completed'
+                    }
+                ]).done(function() {
+                    self.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
```

Comment:
@muhammad-ammar Actually I made a silly mistake and was looking at the wrong browser (!!) It still does not work for me in Firefox (50.1.0) and now with updated OSX 10.11.6.
In the sandbox, when I was able to successfully upload a video file using Chrome, the new upload always shows up in the list with a status of "Pending".